### ISSUE TITLE
Implement Chambers, Mallow and Struck for $\alpha$-stable processes

### ISSUE THREAD

**Issue by venkatachalapathy** [mschauer/Bridge.jl#43]
I am looking to run experiments to calculate first exits across a boundary using Levy-driven SDE. Does this package have a quick and dirty way to simulate $\alpha$-stable distribution based jump SDEs using Bridge.jl?

**Comment by mschauer** [comment]
Do you think of symmetric stable? Then you could use Random walk approximation
by Chambers, Mallow and Struck 
https://pdfs.semanticscholar.org/6cf2/458c48b5d903d35afc21cedc285145de0abb.pdf to generate the driving process. The Euler solver then takes the driving process as argument, I used  it to solve Gamma process driven SDE (Gamma process is implemented)

**Comment by venkatachalapathy** [comment]
Yes. I plan to use your Gamma process model too. Thanks for the tip.  [Nolan's notes](http://fs2.american.edu/jpnolan/www/stable/stable.html) on stable distributions are the one that I usually refer to but could not find anything in Julia.  Thanks for the quick response. 

I am new to Julia but am excited to try it out to teach some of my upcoming courses and use it in my upcoming projects.
